.circleci/config.yml

```diff
@@ -179,49 +132,88 @@ jobs:
+          restore_cache_key_1: spree-brakeman-{{ .Branch }}
+          restore_cache_key_2: spree-brakeman
+          save_cache_key: spree-brakeman-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+          run_file_path: ./bin/brakeman.sh
           requires:
-            - bundle_ruby_3_2
-      - tests_ruby_3_2_rails_7_0_postgres:
+              - bundle_ruby_3_2
+      - tests_postgres:
+          name: tests_ruby_3_2_rails_7_0_postgres
+          rails_version: "~> 7.0.0"
+          ruby_image: cimg/ruby:3.2.2-browsers
+          redis_image: circleci/redis:6.2-alpine
+          postgres_image: circleci/postgres:12-alpine
+          restore_cache_key_1: spree-bundle-v10-ruby-3-2-{{ .Branch }}
+          restore_cache_key_2: spree-bundle-v10-ruby-3-2
+          store_artefacts: true
+          run_file_path: ./bin/tests_postgres.sh
           requires:
-            - bundle_ruby_3_2
-      - tests_ruby_3_2_rails_7_1_postgres:
+              - bundle_ruby_3_2
+      - tests_postgres:
+          name: tests_ruby_3_2_rails_7_1_postgres
+          rails_version: "~> 7.1.0"
+          ruby_image: cimg/ruby:3.2.2-browsers
+          redis_image: circleci/redis:6.2-alpine
+          postgres_image: circleci/postgres:12-alpine
+          restore_cache_key_1: spree-bundle-v10-ruby-3-2-{{ .Branch }}
+          restore_cache_key_2: spree-bundle-v10-ruby-3-2
+          store_artefacts: true
+          run_file_path: ./bin/tests_postgres.sh
           requires:
-            - bundle_ruby_3_2
-      - tests_ruby_3_2_rails_6_1_postgres:
+              - bundle_ruby_3_2
+      - tests_postgres:
+          name: tests_ruby_3_2_rails_6_1_postgres
+          rails_version: "~> 6.1.0"
+          ruby_image: cimg/ruby:3.2.2-browsers
+          redis_image: circleci/redis:6.2-alpine
+          postgres_image: circleci/postgres:12-alpine
+          restore_cache_key_1: spree-bundle-v10-ruby-3-2-{{ .Branch }}
+          restore_cache_key_2: spree-bundle-v10-ruby-3-2
+          store_artefacts: true
+          run_file_path: ./bin/tests_postgres.sh
           requires:
-            - bundle_ruby_3_2
-      - tests_ruby_3_2_rails_7_0_mysql:
+              - bundle_ruby_3_2
+      - tests_mysql:
+          name: tests_ruby_3_2_rails_7_0_mysql
+          rails_version: "~> 7.0.0"
+          ruby_image: cimg/ruby:3.2.2-browsers
+          redis_image: circleci/redis:6.2-alpine
+          mysql_image: circleci/mysql:8-ram
+          restore_cache_key_1: spree-bundle-v10-ruby-3-2-{{ .Branch }}
+          restore_cache_key_2: spree-bundle-v10-ruby-3-2
+          store_artefacts: true
+          run_file_path: ./bin/tests_mysql.sh
           requires:
-            - bundle_ruby_3_2
+              - bundle_ruby_3_2
       - send_test_coverage:
+          name: send_test_coverage
+          rails_version: "~> 7.0.0"
+          ruby_image: cimg/ruby:3.2.2-browsers
+          redis_image: circleci/redis:6.2-alpine
+          run_file_path: ./bin/tests_mysql.sh
```

Comment:
What's the difference between tests_mysql and tests_postgres.sh? It seems like they do almost the same - maybe we could just have `bin/test-ci.sh`?